mne_qt_browser/_pg_figure.py

```diff
@@ -1809,34 +1842,86 @@ def __init__(self, main, title="Settings", **kwargs):
         self.scroll_sensitivity_slider.setValue(self.mne.scroll_sensitivity)
         layout.addRow("horizontal scroll sensitivity", self.scroll_sensitivity_slider)
 
-        # Add subgroup box to show channel type scalings
         layout.addItem(QSpacerItem(10, 10, QSizePolicy.Minimum, QSizePolicy.Expanding))
-        ch_scaling_box = QGroupBox("Channel Type Scalings")
-        ch_scaling_box.setStyleSheet("QGroupBox { font-size: 12pt; }")
-        ch_scaling_layout = QFormLayout()
-        self.ch_scaling_spinboxes = {}
 
-        # Get all unique channel types and allow scaling
+        # Get all unique channel types
         ordered_types = self.mne.ch_types[self.mne.ch_order]
         unique_type_idxs = np.unique(ordered_types, return_index=True)[1]
-        ch_types_ordered = [ordered_types[idx] for idx in sorted(unique_type_idxs)]
-        for ch in ch_types_ordered:
-            if ch in self.mne.unit_scalings.keys():
-                ch_spinbox = QDoubleSpinBox()
-                ch_spinbox.setMinimumWidth(100)
-                ch_spinbox.setRange(0, float("inf"))
-                ch_spinbox.setDecimals(1)
-                ch_spinbox.setStepType(QAbstractSpinBox.AdaptiveDecimalStepType)
-                inv_norm = _get_channel_scaling(self, ch)
-                ch_spinbox.setValue(inv_norm)
-                ch_spinbox.valueChanged.connect(
-                    _methpartial(self._update_spinbox_values, ch_type=ch)
-                )
-                self.ch_scaling_spinboxes[ch] = ch_spinbox
-                ch_scaling_layout.addRow(f"{ch} ({self.mne.units[ch]})", ch_spinbox)
+        ch_types_ordered = [
+            ordered_types[idx]
+            for idx in sorted(unique_type_idxs)
+            if ordered_types[idx] in self.mne.unit_scalings.keys()
```

Comment:
For `dict` you can directly query the dict itself for membership rather than checking for membership in the `.keys()`
```suggestion
            if ordered_types[idx] in self.mne.unit_scalings
```